pkg/transformer/eth_getBalance.go

```diff
@@ -50,19 +50,20 @@ func (p *ProxyETHGetBalance) Request(rawreq *eth.JSONRPCRequest) (interface{}, e
 			return nil, err
 		}
 
+
 		balance := decimal.NewFromFloat(0)
 		for _, utxo := range *qtumresp {
 			balance = balance.Add(utxo.Amount)
 		}
 
 		// 1 QTUM = 10 ^ 8 Satoshi
-		
+		balance = balance.Mul(decimal.NewFromFloat(1e18))
```

Comment:
Any reason why this is needed?

Reply:
No, this can be ignored. I was having a precision issue, but it was due to something else.

Reply:
ok lets take it out then.